idaes/models_extra/gas_distribution/properties/tests/test_natural_gas.py

```diff
@@ -135,6 +135,20 @@ def test_build(self):
             pyo.units.kmol / pyo.units.hr,
         )
 
+        enth_flow = m.state.get_enthalpy_flow_terms("Vap")
+        pred_enth_flow = pyo.value(
+            (293.15 - 298.15) * m.state.cp_mol * m.state.flow_mol
+        )
+        self.assertAlmostEqual(
```

Comment:
Generally comment, but we generally use `assert pytest.approx() == value`  for these cases rather than `unittest`.

Reply:
Muscle memory I guess. I like `pyomo.common.unittest` because I have access to `assertStructuredAlmostEqual`. Don't think I used that anywhere here though.